docs/content_block_manager/adr/0002-use-publishing-api.md

```diff
@@ -8,20 +8,22 @@ The Content Block Manager is a service that allows discrete pieces of content (C
 GOV.UK documents, via an embed code. This enables consistent reuse, and edit-once-update-everywhere behaviour. 
 
 To use a Block, a user has to navigate to the CBM frontend to find the Block they want to use, copy an embed code, and 
-then navigate back to their Document in a publisher app to copy/paste the code into the edit form. 
```

Comment:
Can we squash this into the previous commit?

Reply:
ah yep will do, I just did as a fixup to show what I'd addressed from the chat here https://gds.slack.com/archives/CAB4Q3QBW/p1745414611278879